apps/qubit/modules/informationobject/templates/inventorySuccess.php

```diff
@@ -49,18 +49,7 @@
               <?php endif; ?>
             </td>
             <td>
-              <?php if (isset($doc['dates'])): ?>
-                <?php foreach ($doc['dates']->getRawValue() as $date): ?>
-                  <?php if (isset($date['startDateString'])
-                    || isset($date['endDateString'])
-                    || null != get_search_i18n($date, 'date', array('culture' => $culture))): ?>
```

Comment:
Should you add  to the array `'allowEmpty => 'false'` to prevent return of empty string?